<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QDialog" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(31, 31, 31);
color: white;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>361</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>74</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>74</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>247</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PIC-Simulator </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Ver. 1.0 Mai 2021 </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Dominik Klein, Robin Weisenburger</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>entwickelt im Rahmen des Studiums</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>mit</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
